src/roswire/ros1/state.py

```diff
@@ -21,7 +61,7 @@ class SystemStateProbe:
 
     @classmethod
     def via_xmlrpc_connection(
-        cls, connection: xmlrpc.client.ServerProxy
+            cls, connection: xmlrpc.client.ServerProxy
```

Comment:
Minor nitpick: There's a little bit of excess indentation here, and it would be easier to read if `connection` were on another line.